docs/README-DOCS.md

```diff
@@ -0,0 +1,44 @@
+# Documenting nbconvert
+
+[Documentation for `nbconvert`](https://nbconvert.readthedocs.org/en/latest/)
+is hosted on ReadTheDocs.
+
+## Build Documentation locally
+
+1. Change directory to documentation root:
+
+       $ cd docs
```

Comment:
These aren't showing up as monospace blocks on Github ([view file here](https://github.com/willingc/nbconvert/blob/doc-sphinxone/docs/README-DOCS.md)). It looks like you need one more space of indentation for each:

```
1. Change directory to documentation root:

        $ cd docs
```

becomes:


1. Change directory to documentation root:

        $ cd docs